### PR TITLE
fix patch filename

### PR DESCRIPTION
**Pull request by windflyer**
1. display-pkg-name.patch cannot by applied if we don't fix it.
2. checkFileExtn() should return 0 when apportLength > extnLength is true. Otherwise ``/var/crash/.lock'' will be included.

### CODE REVIEW

File: debian/apport-notifyd.c [hunk 1]
```diff
@@ -23,19 +23,21 @@
...
-	apportLength = strlen(CRASH_EXTN);
-	// syslog(LOG_NOTICE,"extnLength is %d and apportLength is %d\n", extnLength, apportLength);
+    extnLength = strlen(extn);
+    apportLength = strlen(CRASH_EXTN);
+    // syslog(LOG_NOTICE,"extnLength is %d and apportLength is %d\n", extnLength, apportLength);
 
-	if(apportLength > extnLength) return 1;
+    // if a file's name is shorter than ".crash",
+    // apparently it's not a crash dump 
+    if(apportLength > extnLength) return 0;
```
Comment (windflyer) [inline debian/apport-notifyd.c]: Should return 0 here. Otherwise /var/crash/.lock will be included.

Comment (rickysarraf) [inline debian/apport-notifyd.c]: That is odd. apport has never reported me of the .lock file. Have you tested this ?

**Comment by rickysarraf** [comment]
Please rebase against the latest revision in my tree.

File: debian/apport-notifyd.c (hunk 1 above)
Comment (rickysarraf) [inline debian/apport-notifyd.c]: You are right... I checked further.

**Comment by rickysarraf** [comment]
This should be done now, as I did a manual merge.

**Pull request closed without merging**